exporter/loadbalancingexporter/README.md

```diff
@@ -24,7 +24,9 @@ Refer to [config.yaml](./testdata/config.yaml) for detailed examples on using th
 * The `resolver` accepts either a `static` node, or a `dns`. If both are specified, `dns` takes precedence.
 * The `hostname` property inside a `dns` node specifies the hostname to query in order to obtain the list of IP addresses.
 * The `dns` node also accepts an optional property `port` to specify the port to be used for exporting the traces to the IP addresses resolved from `hostname`. If `port` is not specified, the default port 4317 is used.
-
+* The `routing_key`, which only applies to `trace` pipeline types, supports one of the following values:
+    * `service`: routes spans based on its service name.
```

Comment:
What's the use case for this? When would this be useful? In general, the same trace has different services as part of the same transaction. What happens in that case, would spans for the same trace arrive at different collectors, based on the service names?